docs/reference/jinja2.md

```diff
@@ -108,6 +114,12 @@ Or resize an image and retrieve the renditions for more bespoke use:
 <div class="wrapper" style="background-image: image-set(url({{ bg.renditions[0].url }}) 1x, url({{ bg.renditions[1].url }}) 2x);"></div>
 ```
 
+When working with SVG images, you can use the `preserve_svg` in filter string to prevent operations that would require rasterizing the SVG.
```

Comment:
```suggestion
When working with SVG images, you can use `preserve_svg` in the filter string to prevent operations that would require rasterizing the SVG.
```